rmgpy/tools/canteramodel.py

```diff
@@ -208,24 +241,36 @@ def __init__(self, species_list=None, reaction_list=None, canteraFile='', output
         `sensitive_species`: a list of RMG species objects for conductng sensitivity analysis on
         `thermo_SA`: a boolean indicating whether or not to run thermo SA. By default, if sensitive_species is given,
                       only kinetic_SA will be calculated and it must be additionally specified to perform thermo SA.
+        `surface_species_list`: list of RMG species objects contianing the surface species. This is necessary for all surface mechanisms.
+        `surface_reaction_list`: list of RMG reaction objects containing the surface reactions.
```

Comment:
Is this also necessary for all surface mechanisms? (like the above arg)

Reply:
The answer is no. We need it just for species to specify the starting mole fractions. I added a small clause to the docstring to clarify, but let me know if you think it can be made clearer